sprocs/grading_jobs_insert_internal.sql

```diff
@@ -44,10 +44,6 @@ BEGIN
 
     IF NOT FOUND THEN RAISE EXCEPTION 'no such submission_id: %', submission_id; END IF;
 
-    IF grading_method != 'Internal' THEN
-        RAISE EXCEPTION 'grading_method is not Internal for submission_id: %', submission_id;
-    END IF;
-
```

Comment:
We should undo this deletion once we've added the new `sprocs/grading_jobs_insert_manual`